scheds/rust/scx_bpfland/Cargo.toml

```diff
@@ -10,10 +10,10 @@ license = "GPL-2.0-only"
 anyhow = "1.0.65"
 ctrlc = { version = "3.1", features = ["termination"] }
 clap = { version = "4.1", features = ["derive", "env", "unicode", "wrap_help"] }
-libbpf-rs = "0.23"
+libbpf-rs = "0.23.1"
 log = "0.4.17"
 scx_utils = { path = "../../../rust/scx_utils", version = "1.0.1" }
-simplelog = "0.12.0"
+simplelog = "0.12"
```

Comment:
Latest version of `simplelog` is [0.12.2](https://docs.rs/simplelog/latest/simplelog/) is this expected?

Reply:
Oh, yeah, I meant latest among the existing dependencies. I just dropped .0 patch levels.